src/core/issues-notes.js

```diff
@@ -198,12 +199,18 @@ async function fetchAndStoreGithubIssues(githubAPI) {
     .filter(issueNumber => issueNumber)
     .map(async issueNumber => {
       const issueURL = `${githubAPI}/issues/${issueNumber}`;
+      const headers = {
+        // Get back HTML content instead of markdown
+        // See: https://developer.github.com/v3/media/
+        Accept: "application/vnd.github.v3.html+json",
+      };
+      if (githubUser && githubToken) {
+        const credentials = btoa(`${githubUser}:${githubToken}`);
+        const Authorization = `Basic ${credentials}`;
+        Object.assign(headers, { Authorization });
```

Comment:
Short but duplicated, should be make a helper function?

Reply:
Yeah, was thinking the same. However, we need to refactor the github module to be in control of all github related fetches. That way, the credentials can be added when present in the config, and the request count/limit can be centrally managed. That's a larger refactor, however. 

I'll file a separate bug for that. Would that be ok?

Reply:
Yup 👍